libs/libzauth/libzauth/src/zauth.rs

```diff
@@ -85,6 +85,15 @@ impl FromStr for TokenType {
     }
 }
 
+// Token Verification ////////////////////////////////////////////////////////
+
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
```

Comment:
`PartialEq`?  what happens if we remove that?  (i'm clueless with rust)

Reply:
It looks like to derive `Eq` we need to derive `PartialEq` as well.